DevelopGitHub/GitHubExtension/GitHubExtension.DB/Activities.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE TABLE [dbo].[Activities]
+(
+	[Id] INT NOT NULL IDENTITY(1,1), 
```

Comment:
It was all ok when I opened files in visual studio, but when I push it to github, code dancing, the problem was fixed when I open file in notepad and aligned it